src/devicecontroller/VideoQualitySettings.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export default class VideoQualitySettings {
```

Comment:
Do we want to add documentation here, like something we did with [MeetingSessionTURNCredentials](https://github.com/aws/amazon-chime-sdk-js/blob/master/src/meetingsession/MeetingSessionTURNCredentials.ts#L4-L6). This would help customers understand what these settings actually mean.

Reply:
Thought about it but decided against it since it is pretty self-explanatory (width height, etc.)